lib/game.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_relative 'player.rb'
+require_relative 'board.rb'
+
+# Game class
+class Game
+  attr_reader :board_game, :player1, :player2
+  attr_accessor :symbol1, :symbol2
+
+  def initialize
```

Comment:
- [ ] Use attr_readers/attr_writers instead of attr_acessors. By using it you allow writing that property from outside the class. `symbol1` and `symbol2` don't need to be `attr_accessor`. We can also get rid of symbols as it has been already defined from player class.